tests/runtests.py

```diff
@@ -103,13 +116,13 @@ def test_cmd_dereference(self):
         res = gdb_start_silent_cmd("dereference $sp")
         self.assertNoException(res)
         self.assertTrue(len(res.splitlines()) > 2)
-        self.assertIn("$rsp", res)
```

Comment:
Why drop this?

Reply:
Because it would make the `test_cmd_dereference` test specific to x64

Reply:
Hm. Wish there were a way to test that the frame pointer were in the result.

If only we could import gef into the test itself and use `Architecture`

Reply:
Funny you mention that, while working the non x86 unit tests I was thinking we could have a pipe setup between the running test and gdb (a-la-`pwnlib.gdb`) it would be much more flexible because we could extract any info from the running gdb, but honestly I don't want to implement it, that seems overkill to me just for tests.

Reply:
Making an external API to gdb would be great in general. We actually talked about this a bit last year :)